src/controller/CHIPDevice.cpp

```diff
@@ -59,7 +59,7 @@ CHIP_ERROR Device::SendMessage(System::PacketBufferHandle buffer)
     // If there is no secure connection to the device, try establishing it
     if (mState != ConnectionState::SecureConnected)
     {
-        err = LoadSecureSessionParameters();
+        err = LoadSecureSessionParameters(false);
```

Comment:
add ` LoadSecureSessionParameters(false /* resetNeeded */);` 

or maybe even more readable, add an enum so I can call this as:

`LoadSecureSessionParameters(Device::kResetTransport);` or
`LoadSecureSessionParameters(Device::kKeepTransport);` 

to make it easier to read.